packages/markdown/components/CodeTabs/index.jsx

```diff
@@ -21,9 +23,10 @@ const CodeTabs = props => {
       <div className="CodeTabs-toolbar">
         {children.map(({ props: pre }, i) => {
           const { meta, lang } = pre.children[0].props;
+          /* istanbul ignore next */
           return (
             <button key={i} onClick={e => handleClick(e, i)} type="button">
-              {meta || `(${lang || 'plaintext'})`}
+              {meta || `${!lang ? 'Text' : upper(lang) || lang}`}
```

Comment:
`upper(lang)` will always return the `lang` if it's not a recognized language so I don't know if you need the `|| lang` here

Reply:
Let's get rid of it...

```suggestion
              {meta || `${!lang ? 'Text' : upper(lang)}`}
```